build.sbt

```diff
@@ -201,8 +202,8 @@ lazy val V = new {
   def scala2Versions = nonDeprecatedScala2Versions ++ deprecatedScala2Versions
 
   // Scala 3
-  def nonDeprecatedScala3Versions = Seq(scala3, "0.25.0")
-  def deprecatedScala3Versions = Seq("0.25.0-RC2", "0.26.0-RC1")
+  def nonDeprecatedScala3Versions = Seq(scala3, scala3Candidate)
```

Comment:
It seems that we have a large regression in 0.27 and no completions coming from scala library are not displayed.
This way most large tests will be run on 0.26, which works and I will focus on fixing it upstream before the release.

Reply:
Link to the regression https://github.com/lampepfl/dotty/issues/9760